src/Communicator.ts

```diff
@@ -84,14 +83,14 @@ export class Communicator {
      * @param commitDigest The digest hash of the transaction to commit.
      * @returns Promise which fulfills with the commit transaction response returned from QLDB.
      */
-    async commit(txnId: string, commitDigest: CommitDigest): Promise<CommitTransactionResult> {
-        const request: SendCommandRequest = {
+    async commit(txnId: string, commitDigest: Uint8Array | undefined): Promise<CommitTransactionResult> {
```

Comment:
CommitDigest?

Reply:
CommitDigest is not available in [@aws-sdk/client-qldb-session](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-qldb-session/index.html)

Reply:
https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-qldb-session/modules/committransactionrequest.html#commitdigest